packages/modules/create-imodel/src/components/imodel-name/IModelName.tsx

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { LabeledInput } from "@itwin/itwinui-react";
+import React from "react";
+
+import { IModelContext } from "../context/imodel-context";
+
+export type IModelNameProps = {
+  /** iModel name property. */
+  nameString?: string;
```

Comment:
hmm.. not sure about these strings per components.. maybe lets leave whole string object in the root components and get it through context??

Reply:
this is actually label property which will be shown for the html element